site/source/user-manual/connecting/connecting-tor/tor-firefox/torff-android.rst

```diff
@@ -7,45 +7,50 @@ Configuring Firefox for Tor on Android
 .. caution::
   This guide assumes you have completed :ref:`setting up Tor for Android<tor-android>`. Please visit this section before proceeding as it is required for Firefox to properly work with Tor.
 
+.. note:: Setups may strongly vary across different Android forks. We have made specific mention of GrapheneOS which has been tested on Pixel 5 and Pixel 6.
+
 Once Tor is setup on your system, you can proceed to setup Firefox:
 
-1. Download `Firefox Beta <https://play.google.com/store/apps/details?id=org.mozilla.firefox_beta>`_ from the Play Store, or `Fennec <https://f-droid.org/en/packages/org.mozilla.fennec_fdroid/>`_ from F-Droid.
+#. Download `Firefox Beta <https://play.google.com/store/apps/details?id=org.mozilla.firefox_beta>`_ from the Play Store, or `Fennec <https://f-droid.org/en/packages/org.mozilla.fennec_fdroid/>`_ from F-Droid.
+
+#. Next, download a `Proxy Auto Config` file that will use Orbot to resolve `.onion` URLs. We have one hosted `here <https://registry.start9labs.com/sys/proxy.pac>`_.
 
-2. Next, download a `Proxy Auto Config` file that will use Orbot to resolve `.onion` URLs. We have one hosted `here <https://registry.start9labs.com/sys/proxy.pac>`_.
+#. Navigate to ``about:config`` in the Firefox URL bar.
 
-3. Navigate to ``about:config`` in the Firefox URL bar.
+   .. figure:: /_static/images/tor/about_config.png
+     :width: 50%
+     :alt: Firefox about config
 
-  .. figure:: /_static/images/tor/about_config.png
-    :width: 50%
-    :alt: Firefox about config
+#. You are going to have to change a few options in here. First, type ``network.proxy.type`` into the search bar, and set the value to ``2``.
 
-4. You are going to have to change a few options in here. First, type ``network.proxy.type`` into the search bar, and set the value to ``2``.
+   .. figure:: /_static/images/tor/network_proxy_type.png
+     :width: 50%
+     :alt: Firefox network proxy type setting screenshot
 
-  .. figure:: /_static/images/tor/network_proxy_type.png
-    :width: 50%
-    :alt: Firefox network proxy type setting screenshot
+#. (This step is for GrapheneOS only): Head to *Settings -> Apps -> Firefox Beta -> Permissions -> Photos and videos -> Configure Storage Scopes -> ADD FILE* then navigate to where you placed the proxy.pac file.
```

Comment:
Calyx has not added Storage Scopes (maybe someone could ask their team if they have a plan for this), and so currently we just need to say we don't support Calyx for this guide

Reply:
This may be possible without the need for storage scopes on Calyx. It could work just like regular Android.

Reply:
It apparently is not possible on Calyx.  Also not possible on LineageOS v20 (Android 13)

Reply:
ok will mention lineage too